piccolo/columns/column_types.py

```diff
@@ -993,22 +997,59 @@ class Concert(Table):
     timedelta_delegate = TimedeltaDelegate()
 
     def __init__(
-        self, default: TimestamptzArg = TimestamptzNow(), **kwargs
+        self,
+        default: TimestamptzArg = TimestamptzNow(),
+        at_time_zone: ZoneInfo = ZoneInfo("UTC"),
+        **kwargs,
     ) -> None:
         self._validate_default(
             default, TimestamptzArg.__args__  # type: ignore
         )
 
         if isinstance(default, datetime):
-            default = TimestamptzCustom.from_datetime(default)
+            default = TimestamptzCustom.from_datetime(default, at_time_zone)
 
         if default == datetime.now:
```

Comment:
I'm a bit confused about this `if` statement. It seems that `datetime.now` is a callable, which doesn't seem to align with the `TimestamptzArg` from the signature. By the way, it seems that the test doesn't cover this scenario involving a callable.

Reply:
Oops, it should be the line `if default == datetime.now`.